### PR TITLE
Bugfix/new currency fails at harvesine distance

### PR DESCRIPTION
**Pull request by clangenb**
* Fix overflow error  in `encointer_community::harvesine_distance` by changing the type of `Degree` to `I64F64`

### CODE REVIEW

**Comment by clangenb** [comment]
Issue in substrate fixed: https://github.com/encointer/substrate-fixed/issues/6

File: ceremonies/src/lib.rs [hunk 1]
```diff
@@ -627,8 +628,8 @@ impl<T: Trait> Module<T> {
         let perdegree = day / T::Moment::from(360u32);
         let start = next - duration;
         // rounding to the lower integer degree. Max error: 240s = 4min
-        let abs_lon: i32 = mlocation.lon.abs().lossy_into();
-        let abs_lon_time = T::Moment::from(abs_lon as u32) * perdegree;
+        let abs_lon: u32 = i64::lossy_from(mlocation.lon.abs()).saturated_into();
```
Comment (brenzi) [inline ceremonies/src/lib.rs:631]: lon is signed! If unit tests pass, then we need better test coverage!

**Review by brenzi — CHANGES_REQUESTED** [review]

File: ceremonies/src/lib.rs (hunk 1 above)
Comment (clangenb) [reply ceremonies/src/lib.rs:631]: But we are taking the absolute value here?

Comment (brenzi) [reply ceremonies/src/lib.rs:631]: true that

**Pull request merged**